src/analysis/seminorm.lean

```diff
@@ -419,9 +472,27 @@ begin
 end
 
 end norm_one_class
+end module
+end semi_normed_ring
+
+section normed_field
+variables [normed_field 𝕜] [add_comm_group E] [add_comm_group F] [module 𝕜 E] [module 𝕜 F]
+
+lemma comp_smul (p : seminorm 𝕜 F) (f : E →ₗ[𝕜] F) (c : 𝕜) :
+  p.comp (c • f) = (∥c∥.to_nnreal) • (p.comp f) :=
+ext $ λ _, by rw [comp_apply, smul_apply, linear_map.smul_apply, p.smul, nnreal.smul_def,
+  real.coe_to_nnreal _ (norm_nonneg c), smul_eq_mul, comp_apply]
+
+lemma comp_smul_apply (p : seminorm 𝕜 F) (f : E →ₗ[𝕜] F) (c : 𝕜) (x : E) :
+  p.comp (c • f) x = ∥c∥ * p (f x) := p.smul _ _
```

Comment:
I made two lemmas here, the first one is a bit useless, because half of the proof is to pack it again into boilerplate structure. I am not sure whether it should stay.